fenzo-core/src/test/java/com/netflix/fenzo/ShortfallAutoscalerTest.java

```diff
@@ -250,7 +250,6 @@ public void testShortfallScaleup2groups() throws Exception {
         int waitingFor = 2;
         while (waitingFor > 0) {
             final Map<String, Integer> map = scaleupActionsQ.poll(coolDownSecs, TimeUnit.SECONDS);
-            Assert.assertNotNull(map);
```

Comment:
I thought that this needed adjusting but it looks like this assertion occasionally fails on `master` too.